.gitignore

```diff
@@ -26,6 +28,7 @@ yarn-error.log*
 .pnpm-debug.log*
 
 # local env files
+.env
```

Comment:
We can use `.env.local` file instead 😄

Reply:
Removed https://github.com/Lahphim/react-surveyor-challenge/pull/47/commits/638716af51845a082db40052d157dc9c56c08ecd